packages/tools/devtools/devtools-view/src/components/SettingsView.tsx

```diff
@@ -66,7 +72,11 @@ export function SettingsView(props: SettingsProps): React.ReactElement {
 			<label style={{ fontSize: "12px" }}>Select theme</label>
 			<Dropdown
 				placeholder="Theme"
-				style={{ minWidth: "150px", fontWeight: "bold" }}
+				style={{
+					minWidth: "150px",
+					fontWeight: "bold",
+				}}
+				appearance="filled-darker"
```

Comment:
I don't think this is the appropriate solution. There are more dropdowns in other views, and having to add these bit to all dropdowns we use everywhere isn't the most maintainable thing.

From the styles we're currently overriding it sounds like 'colorCompoundBrandStroke` (set to black) might be the cause of this... have you tried removing that one? You'd know better than anyone if it affected something else though.

Reply:
(Plus the lack of borders when the component and the background are the same color just feels like it makes the UI harder to use)